definitions/variable/industry/tag_nonfer_commodities.yaml

```
@@ -0,0 +1,13 @@
+- Non-Ferrous Metals Commodity:
+    - Aluminum:
+        description: Aluminum, commonly produced by electrolysis of alumina or re-melting of scrap
+    - Secondary Aluminum:
+        description: Aluminum produced from aluminum scrap recycling
+    - Primary Aluminum:
+        description: Aluminum produced via Hall-Heroult process from alumina
+    - Alumina:
```

Suggest to rename to avoid confusion..
```suggestion
    - Aluminum oxide:
```